blueman/main/indicators/StatusNotifierItem.py

```diff
@@ -36,7 +36,11 @@ def _get_layout(self, parent_id: int, _recursion_depth: int, _property_names: Li
         if parent_id == 0:
             return self._revision, (0, {}, self._render_menu(self._items, 1, self._render_submenu))
         else:
-            return self._revision, (parent_id, self._render_item(self._items[parent_id - 1]), [])
+            item = self._items[parent_id - 1]
+            if "submenu" in item:
+                return self._revision, (parent_id, {}, self._render_submenu(item, parent_id ))
```

Comment:
Are you sure `{}` is correct here? Shouldn't it just be
```
return self._revision, (parent_id, self._render_item(self._items[parent_id - 1]), self._render_submenu(item, parent_id))
```
for all `parent_id != 0`?

Reply:
Yes, you're right.  The audio profile menu had disappeared with that change.

Reply:
There's no need for the `else` branch then as `_render_submenu` handles items without submenus just fine.

Also there's a whitespace error that makes mypy unhappy.

Reply:
Thanks, I have changed this and added a check for the _recursion_depth parameter